Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -1453,4 +1453,8 @@ lemma integrable_prod {f : α → E × F} :
     Integrable f μ ↔ Integrable (fun x ↦ (f x).1) μ ∧ Integrable (fun x ↦ (f x).2) μ :=
   ⟨fun h ↦ ⟨h.fst, h.snd⟩, fun h ↦ h.1.prod_mk h.2⟩
 
+lemma Integrable.iff_ofReal {X : Type*} [MeasurableSpace X] {μ : Measure X}
+    {f : X → ℝ} : Integrable f μ ↔ Integrable (fun x ↦ (f x : ℂ)) μ :=
+    ⟨fun hf ↦ hf.ofReal, fun hf ↦ hf.re⟩
```

Comment:
It's better to reuse types which are already introduced in a file for consistency. Can you put this below `Integrable.im`?
```suggestion
lemma Integrable.iff_ofReal {f : α → ℝ} : Integrable f μ ↔ Integrable (fun x ↦ (f x : ℂ)) μ :=
  ⟨fun hf ↦ hf.ofReal, fun hf ↦ hf.re⟩
```

Reply:
fixed